bundle/config/mutator/translate_paths.go

```diff
@@ -117,6 +128,9 @@ func (m *translatePaths) translateJobTask(dir string, b *bundle.Bundle, task *jo
 
 	if task.SparkPythonTask != nil {
 		err = m.rewritePath(dir, b, &task.SparkPythonTask.PythonFile, m.translateFilePath)
+		if target := (&ErrIsNotebook{}); errors.As(err, target) {
+			return fmt.Errorf("please use notebook task type for notebooks. %s", target)
```

Comment:
If we assume the intent to be to use a Python task, the error should state the reverse, e.g.

`expected a file for "task.spark_python_task.python_file" but got a notebook: %w`